contracts/MyToken_721.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+
+contract MyToken_721 is ERC721, Ownable {
+
+    using Counters for Counters.Counter;
+
+    Counters.Counter private _tokenIdCounter;
+
+    mapping(address => uint) private minters; 
+
+    uint constant TOKEN_PRICE = 10 ** 16 wei;

```

Comment:
can we have token price as constructor parameter ? Let's add this to .env file and load in deployment script.